kentikapi/errors/errors.go

```diff
@@ -0,0 +1,231 @@
+package kentik_errors
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type Code int
+
+const (
+	AuthError          Code = 0
+	InvalidRequest     Code = 1
+	NotFound           Code = 2
+	RateLimitExhausted Code = 3
+	Temporary          Code = 4
+	Timeout            Code = 5
+)
+
+type KentikError struct {
```

Comment:
Please add NewError(code, msg) constructor for KentikError and use for creating new errors in the code.

Reply:
Done